### PR TITLE
check engine health after backup

### PR DESCRIPTION
**Pull request by emesika**
Add a health check after backup to delay tests until engine is healthy

### CODE REVIEW

File: basic-suite-master/test-scenarios/test_002_bootstrap.py [hunk 1]
```diff
@@ -1160,6 +1160,12 @@ def test_verify_engine_backup(ansible_engine, engine_api, ost_dc_name, is_node_s
         '--otopi-environment=OVESETUP_SYSTEM/memCheckEnabled=bool:False'
     )
 
+    # verify engine health after backup
+    url = 'https://{}/ovirt-engine/services/health'.format(engine_fqdn)
```
Comment (tinez) [inline basic-suite-master/test-scenarios/test_002_bootstrap.py:1164]: We can use an f-string here.

File: basic-suite-master/test-scenarios/test_002_bootstrap.py [hunk 2]
```diff
@@ -1160,6 +1160,12 @@ def test_verify_engine_backup(ansible_engine, engine_api, ost_dc_name, is_node_s
...
+    assert assert_utils.equals_within_short(
+        lambda: engine_download(url), b"DB Up!Welcome to Health Status!", [(shell.ShellError)]
```
Comment (tinez) [inline basic-suite-master/test-scenarios/test_002_bootstrap.py:1166]: I'd add `allowed_exceptions=` before that shell error exception - it's better than trying to figure out what this parameter does. Also, we don't need `()` surrounding `shell.ShellError`.

**Review by tinez — COMMENTED** [review]
Apart from minor nits, looks good. I'm tempted to remove that 10s sleep in `test_upload_cirros_image` that comes straight after engine backup... WDYT @bennyz ?

**Comment by tinez** [comment]
@emesika could you please remove that sleep in this PR too?

File: basic-suite-master/test-scenarios/test_002_bootstrap.py (hunk 1 above)
Comment (emesika) [reply basic-suite-master/test-scenarios/test_002_bootstrap.py:1164]: will do

**Pull request merged**